src/app/campaign/campaign.component.ts

```diff
@@ -102,10 +102,11 @@ export class CampaignComponent implements OnInit, OnDestroy {
   createFlight() {
     this.campaignStoreService.campaignFirst$.subscribe(state => {
       const flightId = Date.now();
+      const date = new Date(flightId);
       const flight = {
         name: 'New Flight ' + (Object.keys(state.flights).length + 1),
-        startAt: new Date().toISOString(),
-        endAt: new Date().toISOString(),
```

Comment:
flight start date === flight end date is invalid, so I put in more sensible defaults, today thru the 1st of next month, which includes allocation dates thru the end of the month but not the 1st